src/app/components/dashboard-company-selectbox/dashboard-company-selectbox.coffee

```diff
@@ -60,9 +60,8 @@ DashboardCompanySelectboxCtrl = ($scope, $state, $stateParams, $uibModal, MnoeCu
   #====================================
   # Post-Initialization
   #====================================
-  $scope.$watch MnoeOrganizations.getSelectedId, (val) ->
-    if val?
-      selectOrganization()
+  $scope.$watch MnoeOrganizations.getSelected, (val) ->
+    selectOrganization()
```

Comment:
Can we remove the val parameter if it's not used anymore?

Reply:
The `watch` does not run in the parameter is not provided. (`getSelected` is a method on `MnoeOrganizations` service)